src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  font-size: 13px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.udohr {
+  font-size: 17px;
+  margin: 0;
+}
+
+.lol {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.kek {
+  margin: 0;
+}
+
+.cheburek {
```

Comment:
??? Don't name classes like this. They always need to be meaningful.

Reply:
got it!